converter/src/main/java/gov/cms/qpp/conversion/decode/QualityMeasureIdDecoder.java

```diff
@@ -39,7 +39,7 @@ protected DecodeResult internalDecode(Element element, Node thisNode) {
 		List<String> measureGuids = getMeasureGuid(element);
 
 		measureGuids.forEach(measureGuid ->
-			thisNode.putValue(MEASURE_ID, measureGuid, false));
+			thisNode.putValue(MEASURE_ID, measureGuid.toLowerCase(), false));
```

Comment:
What's the reason for the case change if we're ignoring case when performing the comparison?

Reply:
The reason is because we perform a map check in `QualityMeasureIdValidtor.java` based on this value compared to the `measureData.json` if the eMeasureId doesn't completely match, it doesn't find the correct Uuid to validate. The ignore case validation is strictly for the measure data's sub population uuid's i.e. DENOM, DENEX, IPOP, etc...

Reply:
You can see the validation in the method `QualityMeasureIdValidator.validateMeasureConfigs`